maven-failsafe-plugin/src/main/java/org/apache/maven/plugin/failsafe/VerifyMojo.java

```diff
@@ -139,6 +139,12 @@
     @Parameter( property = "failIfNoTests" )
     private Boolean failIfNoTests;
 
+    /**
+     * The number of flakes after which the overall test suite will be considered a failure.
+     */
+    @Parameter( property = "surefire.failOnFlakeCount", defaultValue = "0" )
+    private int failOnFlakeCount;
```

Comment:
Could we clarify if this is total number of flakes per build (is that what "overall" refers to?) or per test (i.e. two test methods are both retried 3 times, does the build fail for `failOnFlakeCount = 3` or `failOnFlakeCount = 6`)?

Reply:
Exactly this was not clear to me when we spoke with the JUnit5 colleagues several days ago.
The logic of `re-run` feature operates at the level of individual Test.
In this PR I guess we are talking about the level of Test Set (cumulative number of flakes) which is different view from ours. But's that's ok if it would be properly clarified.

Reply:
It's very clear now, thanks!